app/Caster/UnitDiscreteSquawkRangeRuleCaster.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Caster;
+
+use InvalidArgumentException;
+use App\Rules\UnitDiscreteSquawkRange\FlightRules;
+use App\Rules\UnitDiscreteSquawkRange\UnitType;
+use Illuminate\Contracts\Database\Eloquent\CastsAttributes;
+use Illuminate\Contracts\Validation\Rule;
+
+class UnitDiscreteSquawkRangeRuleCaster implements CastsAttributes
+{
+
+    /**
+     * @param \Illuminate\Database\Eloquent\Model $model
+     * @param string $key
+     * @param mixed $value
+     * @param array $attributes
+     * @return Rule
+     */
+    public function get($model, string $key, $value, array $attributes)
```

Comment:
Can you type-hint the model instead of the docstring? (some would say they are the same thing, I just noticed because it was the odd one out).

Reply:
Sadly not - it's a Laravel interface here, but I've updated the docblock to have the correct class.